extensions/json.py

```
@@ -19,7 +19,9 @@
 # JSON -> CLASS
 class namespace:
     # CLASS -> INIT
-    async def init(self, request: Request, response: Response) -> namespace: return self
+    async def init(self, socket: æ.WebSocket) -> namespace: return self
     # CLASS -> CONVERT
     def parse(self, data: dict, keys: list[str]) -> None:
-        for key in [key for key in keys if key in data and isinstance(data[key], str)]: data[key] = loads(data[key])
\ No newline at end of file
+        try: 
+            for key in [key for key in keys if key in data and isinstance(data[key], str)]: data[key] = loads(data[key])
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)